src/game/etj_command_parser.cpp

```diff
@@ -91,12 +90,12 @@ void ETJump::CommandParser::expectMultipleTokens(
   }
 
   if (_current != end(_args)) {
-    // we peeked the next one and we want the parser to continue from
+    // we peeked the next one, and we want the parser to continue from
```

Comment:
Literal nitpicking but the comma is not necessary for this to be valid grammar 😅